notes/SDLC.md

```diff
@@ -0,0 +1,54 @@
+# SDLC 
```

Comment:
it be more appropriate to fully enumerate "SDLC" because students won't likely be familiar with the acronym

Reply:
fixed